SoftLayer/CLI/vpn/ipsec/update.py

```diff
@@ -0,0 +1,102 @@
+"""Updates an IPSEC tunnel context."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI.exceptions import CLIHalt
+
+
+@click.command()
+@click.argument('context_id', type=int)
+@click.option('--friendly-name',
+              default=None,
+              help='Friendly name value')
+# todo: Update to utilize custom IP address type
+@click.option('--remote-peer',
+              default=None,
+              help='Remote peer IP address value')
+@click.option('--preshared-key',
+              default=None,
+              help='Preshared key value')
+@click.option('--p1-auth',
+              '--phase1-auth',
+              default=None,
+              type=click.Choice(['MD5', 'SHA1', 'SHA256']),
+              help='Phase 1 authentication value')
+@click.option('--p1-crypto',
```

Comment:
I'd like most of these options to have defaults where possible, mirroring what is in the control portal for defaults would be good.